ui/tutorial-renderer/src/main/res/layout/tutorial_tips_learn.xml

```diff
@@ -19,43 +19,13 @@
         <include layout="@layout/tutorial_include_margins" />
 
         <include layout="@layout/tutorial_tips_include_positioning" />
-
-        <com.airbnb.lottie.LottieAnimationView
-            android:id="@+id/animation"
-            android:layout_width="0dp"
-            android:layout_height="0dp"
-            app:layout_constraintBottom_toBottomOf="@id/imagePosition"
-            app:layout_constraintEnd_toEndOf="@id/imagePosition"
-            app:layout_constraintStart_toStartOf="@id/imagePosition"
-            app:layout_constraintTop_toTopOf="@id/imagePosition"
-            app:lottie_autoPlay="true"
-            app:lottie_rawRes="@raw/anim_tutorial_tips_people"
-            app:lottie_loop="true" />
-
-        <TextView
-            android:id="@+id/headline"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:breakStrategy="balanced"
-            android:gravity="center_horizontal"
-            android:text="@string/tutorial_tips_learn_headline"
-            android:textAppearance="@style/TextAppearance.GodTools.Tutorial.Tips.Headline"
-            app:layout_constraintEnd_toStartOf="@id/marginEnd"
-            app:layout_constraintStart_toEndOf="@id/marginStart"
-            app:layout_constraintTop_toTopOf="@id/headlinePosition" />
-
-        <TextView
-            android:id="@+id/text"
+        <androidx.compose.ui.platform.ComposeView
+            android:id="@+id/compose"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
-            android:breakStrategy="balanced"
-            android:gravity="center_horizontal"
-            android:text="@string/tutorial_tips_learn_text"
-            android:textAppearance="@style/TextAppearance.GodTools.Tutorial.Text"
-            app:layout_constraintEnd_toStartOf="@id/marginEnd"
-            app:layout_constraintStart_toEndOf="@id/marginStart"
-            app:layout_constraintTop_toBottomOf="@id/headlineBottom" />
+            app:layout_constraintStart_toStartOf="@id/marginStart"
+            app:layout_constraintEnd_toEndOf="@id/marginEnd"
+            app:layout_constraintTop_toTopOf="parent" />
```

Comment:
This view doesn't have a bottom constraint specified, which is probably part of the reason that the content is pushed down as far as it is. try adding the attribute `app:layout_constraintBottom_toTopOf="@id/action_next"`, that will constrain the bottom of the ComposeView to be aligned with the top of the continue button

Reply:
you will also want to change `android:layout_height` to be `0dp` so that it defers the height to be defined by the available space between the top and bottom constraints.